### DISCUSSION TITLE
Background color. I changed background color to black but text not able to see, how to make font color to white

### DISCUSSION THREAD

**Discussion by Narendra678** [apache/superset#16207] [Q&A / Help]
## Screenshot
![Capture](https://user-images.githubusercontent.com/88739186/128974656-98f4e2d4-fd09-45df-8812-9007b154d1f1.PNG)

[drag & drop image(s) here!]

## Description

[describe the issue here!]
Background color. I changed background color to black but text not able to see, how to make font color to white and need to remove white lines.

## Design input
[describe any input/collaboration you'd like from designers, and
tag accordingly. For design review, add the
label `design:review`. If this includes a design proposal,
include the label `design:suggest`]

**Comment by junlincc** [comment]
we don't currently support this feature and i'm not sure if we can change the dashboard properties to achieve the goal? Echarts(our main charting library does offer dark mode but it doesn't cover table and pivot table. If anyone knows how, please let us know 🙏

**Comment by rusackas** [comment]
You basically have to add custom CSS selectors for every chart type that you care about... the text is in all sorts of different elements (TD tags, SVG tags, etc) - then there will also be grid lines and tick marks and stuff that you'll want to make white as well. It can be done, but it's a very intricate task.
